test/perf/PerfGemm.cpp

```diff
@@ -2,14 +2,18 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-//===================-- PerfGemm.hpp - Simple performance tests -=============//
+//===================-- PerfGemm.cpp - Simple performance tests -=============//
 //
 // Copyright 2022 The IBM Research Authors.
 //
 // =============================================================================
 //
-// This file contains tests for simple test cases, like for an arbitrary small
+// This file contains tests for simple test cases, for an arbitrary small
```

Comment:
Remove the comma please. What it does is introduces an unnatural pause or breath into the middle of the sentence that might not otherwise be present. It does make sense if `like` is included, but is otherwise a bit awkward.

Reply:
Thanks, updated in both files where the same comment was written.